src/Result/Store.php

```diff
@@ -6,4 +6,153 @@
 
 class Store extends AbstractResult
 {
+    public function getName(): string
+    {
+        $data = $this->getData();
+        return $data['name'];
+    }
+
+    public function getWebsite(): string
+    {
+        $data = $this->getData();
+        return $data['website'];
+    }
+
+    public function getDefaultCurrency(): string
+    {
+        $data = $this->getData();
+        return $data['defaultCurrency'];
+    }
+
+    public function getInvoiceExpiration(): int
+    {
+        $data = $this->getData();
+        return $data['invoiceExpiration'];
+    }
+
+    public function getMonitoringExpiration(): int
+    {
+        $data = $this->getData();
+        return $data['monitoringExpiration'];
+    }
+
+    public function getSpeedPolicy(): string
+    {
+        $data = $this->getData();
+        return $data['speedPolicy'];
+    }
+
+    public function getLightningDescriptionTemplate(): string
+    {
+        $data = $this->getData();
+        return $data['lightningDescriptionTemplate'];
+    }
+
+    public function getPaymentTolerance(): int
+    {
+        $data = $this->getData();
+        return $data['paymentTolerance'];
+    }
+
+    public function anyoneCanCreateInvoice(): bool
+    {
+        $data = $this->getData();
+        return $data['anyoneCanCreateInvoice'];
+    }
+
+    public function requiresRefundEmail(): bool
+    {
+        $data = $this->getData();
+        return $data['requiresRefundEmail'];
+    }
+
+    public function lightningAmountInSatoshi(): bool
+    {
+        $data = $this->getData();
+        return $data['lightningAmountInSatoshi'];
+    }
+
+    public function lightningPrivateRouteHints(): bool
+    {
+        $data = $this->getData();
+        return $data['lightningPrivateRouteHints'];
+    }
+
+    public function onChainWithLnInvoiceFallback(): bool
+    {
+        $data = $this->getData();
+        return $data['onChainWithLnInvoiceFallback'];
+    }
+
+    public function redirectAutomatically(): bool
+    {
+        $data = $this->getData();
+        return $data['redirectAutomatically'];
+    }
+
+    public function showRecommendedFee(): bool
+    {
+        $data = $this->getData();
+        return $data['showRecommendedFee'];
+    }
+
+    public function getRecommendedBlockTarget(): int
+    {
+        $data = $this->getData();
+        return $data['recommendedBlockTarget'];
```

Comment:
should be `recommendedFeeBlockTarget` `Fee` missing

Reply:
nice catch, fixed